MonoGame.Framework/Platform/Native/Storage.Interop.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Xna.Framework.Storage
+{
+    public sealed partial class StorageDevice
```

Comment:
What is this for?  The `.Interop.cs` files are reserved for the C# to C++ APIs which are all `internal` types.

See https://github.com/MonoGame/MonoGame/blob/develop/MonoGame.Framework/Platform/Native/Graphics.Interop.cs

Reply:
Really you can remove this if you don't plan to write the code here to reference it.

Reply:
Yes remove this...  this isn't how the `.Interop.cs` files are used.